### ISSUE TITLE
[torrentgalaxy] The SSL connection could not be established, see inner exception.: certificate validation failed: [Subject]  E=protect@DDoS-Filter.domain, CN=DDoS-Filter.domain, OU=DDoS-Filter, O=DDoS-Filter, L=DDoS-Filter, S=DDoS-Filter, C=AA[Issuer]  E=protect@DDoS-Filter.domain, CN=protect@DDoS-Filter.domain, OU=CA, O=DDoS-Filter, L=DDoS-Filter, S=DDoS-Filter, C=AA[Serial Number]  00B221528E9B7970ED[Not Before]  1/8/2018 5:41:26 AM[Not After]  10/8/2027 6:41:26 AM[Thumbprint]  194084E1EDCD95C2C8DD368BC1D5094B22BB8665 (Config)

### ISSUE THREAD

**Issue by ondrovic** [Jackett/Jackett#11285]
### Environment

**OS**: CoreMacOs

**.Net Runtime**: .Net-Core

**.Net Version**:

**Jackett Version**:  v0.17.668

**Last Working Jackett Version**:  unknown just tried to add indexer

**Are you using a proxy or VPN?** yes, but tried with both on and off

### Description
When trying to add indexer, this is similar to [11160](https://github.com/Jackett/Jackett/issues/11160) but slightly different

1. Click Add Indexer
2. Search: TorrentGalaxy
3. Click the + button

### Logged Error Messages

```
System.Net.Http.HttpRequestException: The SSL connection could not be established, see inner exception.
 ---> System.Exception: certificate validation failed: [Subject]
  E=protect@DDoS-Filter.domain, CN=DDoS-Filter.domain, OU=DDoS-Filter, O=DDoS-Filter, L=DDoS-Filter, S=DDoS-Filter, C=AA

[Issuer]
  E=protect@DDoS-Filter.domain, CN=protect@DDoS-Filter.domain, OU=CA, O=DDoS-Filter, L=DDoS-Filter, S=DDoS-Filter, C=AA

[Serial Number]
  00B221528E9B7970ED

[Not Before]
  1/8/2018 5:41:26 AM

[Not After]
  10/8/2027 6:41:26 AM

[Thumbprint]
  194084E1EDCD95C2C8DD368BC1D5094B22BB8665

   at Jackett.Common.Utils.Clients.HttpWebClient2.ValidateCertificate(HttpRequestMessage request, X509Certificate2 certificate, X509Chain chain, SslPolicyErrors sslPolicyErrors) in /Users/runner/work/1/s/src/Jackett.Common/Utils/Clients/HttpWebClient2.cs:line 50
   at System.Net.Http.ConnectHelper.<>c__DisplayClass3_0.<EstablishSslConnectionAsync>b__0(Object sender, X509Certificate certificate, X509Chain chain, SslPolicyErrors sslPolicyErrors)
   at System.Net.Security.SecureChannel.VerifyRemoteCertificate(RemoteCertificateValidationCallback remoteCertValidationCallback, ProtocolToken& alertToken, SslPolicyErrors& sslPolicyErrors, X509ChainStatusFlags& chainStatus)
   at System.Net.Security.SslStream.CompleteHandshake(ProtocolToken& alertToken, SslPolicyErrors& sslPolicyErrors, X509ChainStatusFlags& chainStatus)
   at System.Net.Security.SslStream.ForceAuthenticationAsync[TIOAdapter](TIOAdapter adapter, Boolean receiveFirst, Byte[] reAuthenticationData, Boolean isApm)
   at System.Net.Http.ConnectHelper.EstablishSslConnectionAsyncCore(Boolean async, Stream stream, SslClientAuthenticationOptions sslOptions, CancellationToken cancellationToken)
   --- End of inner exception stack trace ---
   at System.Net.Http.ConnectHelper.EstablishSslConnectionAsyncCore(Boolean async, Stream stream, SslClientAuthenticationOptions sslOptions, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.GetHttpConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.HttpConnectionPool.SendWithRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)
   at System.Net.Http.DecompressionHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at System.Net.Http.DiagnosticsHandler.SendAsyncCore(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
   at FlareSolverrSharp.ClearanceHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
   at System.Net.Http.HttpClient.SendAsyncCore(HttpRequestMessage request, HttpCompletionOption completionOption, Boolean async, Boolean emitTelemetryStartStop, CancellationToken cancellationToken)
   at Jackett.Common.Utils.Clients.HttpWebClient2.Run(WebRequest webRequest) in /Users/runner/work/1/s/src/Jackett.Common/Utils/Clients/HttpWebClient2.cs:line 170
   at Jackett.Common.Utils.Clients.WebClient.GetResultAsync(WebRequest request) in /Users/runner/work/1/s/src/Jackett.Common/Utils/Clients/WebClient.cs:line 207
   at Jackett.Common.Indexers.BaseWebIndexer.RequestWithCookiesAsync(String url, String cookieOverride, RequestType method, String referer, IEnumerable`1 data, Dictionary`2 headers, String rawbody, Nullable`1 emulateBrowser) in /Users/runner/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 496
   at Jackett.Common.Indexers.CardigannIndexer.GetConfigurationForSetup(Boolean automaticlogin) in /Users/runner/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 843
   at Jackett.Common.Indexers.CardigannIndexer.DoLogin() in /Users/runner/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 514
   at Jackett.Common.Indexers.CardigannIndexer.ApplyConfiguration(JToken configJson) in /Users/runner/work/1/s/src/Jackett.Common/Indexers/CardigannIndexer.cs:line 919
   at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in /Users/runner/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 97
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /Users/runner/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 61

```
### Screenshots
![image](https://user-images.githubusercontent.com/3682042/110702396-74e80a00-81c0-11eb-8d3d-3ec668a409de.png)
![image](https://user-images.githubusercontent.com/3682042/110702472-8c26f780-81c0-11eb-9a25-c4ca7f893ea3.png)

**Comment by ilike2burnthing** [comment]
Can you access the site in browser from the same device and IP?

Click the configuration (wrench) button instead and try adding the captcha or changing the sitelink.

**Comment by ondrovic** [comment]
1. Yes access can be achieved via the browser by the same machine running Jackett, both off and on VPN.
2. In Version v0.17.668, there is no option for Captcha for TorrentGalaxy.

UPDATE: I updated this morning to Version v0.17.671, which does have the Captcha option, I was able to successfully add TorrentGalaxy as a indexer, once Captcha was filled out. It seems like the new Version v0.17.671 may have fixed the issue.
